doc/source/ray-core/tasks/fault-tolerance.rst

```diff
@@ -8,7 +8,10 @@ because the process crashed or because the machine failed, Ray will rerun
 the task until either the task succeeds or the maximum number of retries is
 exceeded. The default number of retries is 3 and can be overridden by
 specifying ``max_retries`` in the ``@ray.remote`` decorator. Specifying -1
-allows infinite retries, and 0 disables retries.
+allows infinite retries, and 0 disables retries. To override the default number
+of retries for all tasks submitted, set the OS environment variable
+``RAY_TASK_MAX_RETRIES``. e.g., by passing this to your driver script or by
+using :ref:`runtime environments<runtime-environments>`.
```

Comment:
Why don't we have a short example here? 
```
RAY_TASK_MAX_RETRIES driver.py

or 

# in driver.py
ray.init(address="auto", runtime_env={...})
```

Reply:
I'm not sure this is necessary (runtime envs page provides examples) and it kind of breaks the flow of this docs page.